src/dynamicTypes/eventHandlersService.ts

```diff
@@ -9,15 +9,26 @@ export interface EventHandlersService {
   getComponentEventHandlers: (
     componentName: string
   ) => EventHandler[] | undefined;
+  registerComponentEventHandlers: (
+    componentName: string,
+    handlers: EventHandler[]
+  ) => void;
 }
 
-const eventHandlers = eventHandlersJSON as ComponentsEventHandlers;
+const $wEventHandlers = eventHandlersJSON as ComponentsEventHandlers;
+const eventHandlers = { ...$wEventHandlers };
```

Comment:
why like this?

Reply:
How would you do it? @tombenezra

Reply:
discussed in private, might need to convert the eventHandlers json to .ts file in the future, if it will be required.